app/pages/lab/apply-for-beta-form.jsx

```diff
@@ -8,28 +8,17 @@ const MINIMUM_SUBJECT_COUNT = 100;
 const REQUIRED_PAGES = ['Research', 'FAQ'];
 
 // Static functions
-const projectHasActiveWorkflows = (workflows) => {
-  return workflows.some((workflow) => {
-    return workflow.active;
-  });
+const projectHasActiveWorkflows = (project) => {
+  if (project.links.active_workflows && project.links.active_workflows.length > 0) {
+    return true;
+  }
+  return false;
```

Comment:
This function could just `return (project.links.active_workflows && project.links.active_workflows.length > 0)`

Reply:
Done, thanks.